cabal-helper.cabal

```diff
@@ -56,16 +56,22 @@ library
                        Paths_cabal_helper
   ghc-options:         -Wall
   build-depends:       base          < 5    && >= 4.5
-                     , Cabal         < 2.1  && >= 2.0     || < 1.26 && >= 1.14
+  if os(windows)
+    build-depends:     base          < 5    && >= 4.7
+  build-depends:       Cabal         < 2.1  && >= 2.0     || < 1.26 && >= 1.14
                      , directory     < 1.4  && >= 1.1.0.2
                      , filepath      < 1.5  && >= 1.3.0.0
                      , transformers  < 0.6  && >= 0.3.0.0
                      , mtl           < 2.3  && >= 2.0
                      , process       < 1.7  && >= 1.1.0.1
-                     , unix          < 2.8  && >= 2.5.1.1
-                     , semigroupoids < 5.3  && >= 5.2.1
+  if os(windows)
+    build-depends:     unix-compat
```

Comment:
Add lower and upper bounds to this one though. An underapproximation like `< 0.5 && >= 0.4.3.1` should do to start with.